module2/homework/addEven/addEven.hpp

```diff
@@ -4,5 +4,12 @@
 int addEven(const std::vector<int>& numbers) {
     // TODO: Your implementation goes here
     // Below return is only to make this function compile now
-    return -1;
+    int sum {};
+
+    for(const auto& element : numbers){
+        if(!(element % 2)){
```

Comment:
Implicit casts are usually a code smell.
Comparison to 0 is much clearer
```suggestion
        if(element % 2 == 0) {
```